### PR TITLE
Load more functionality

### PR DESCRIPTION
**Pull request by Devemmy01**
I added a load more functionality using vanilla JS

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -318,4 +337,22 @@ ul#contributors .links {
   display: flex;
   justify-content: center;
   gap: .25rem;
-}
\ No newline at end of file
+}
+/* .row{
```
Comment (VectorChi) [inline style.css]: @Devemmy01 kindly clean this part(commented code blocks) up

Comment (Devemmy01) [reply style.css]: I'll do that now

**Comment by Devemmy01** [comment]
@VectorChi I've fixed it

**Comment by Devemmy01** [comment]
Thanks for merging my PR

**Comment by Devemmy01** [comment]
I have another suggestion if you don't mind.
scrolling all the way to the top can be a hectic task for some users. how about I add a scroll to top button?

**Pull request merged**